docs/examples/academy/14. Working with DataFrames.ipynb

```diff
@@ -781,7 +781,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.9"
+   "version": "3.8.5"
```

Comment:
Using the `venv` described in the https://github.com/VorTECHsa/python-sdk/blob/master/CONTRIBUTING.md guide ensures that a consistent python version is used.